docs/community/bindings.md

```diff
@@ -64,3 +64,8 @@ As a C library, bindings can be made to call H3 functions from different program
 ## Ruby
 
 - [StuartApp/h3_ruby](https://github.com/StuartApp/h3_ruby)
+
+## Rust
```

Comment:
I don't think this was intended to be in this PR - maybe needs a rebase?

Reply:
fixed!